src/transactions/PathPaymentStrictReceiveOpFrame.cpp

```diff
@@ -47,7 +47,7 @@ PathPaymentStrictReceiveOpFrame::doApply(AbstractLedgerTxn& ltx)
         return false;
     }
     innerResult().success().last = SimplePaymentResult(
-        getDestID(), getDestAsset(), mPathPayment.destAmount);
+        getDestMuxedAccount(), getDestAsset(), mPathPayment.destAmount);
```

Comment:
This change isn't mentioned in the proposal. Is this change actually desired?

Reply:
yeah I saw this too. I don't think we can actually do this either as SDKs are probably using the return the RAW result (like exchanges checking if a transaction is for them).
This is not a conversation for the PR but against the CAP, so let's discuss elsewhere.

Reply:
Oh it was actually mentioned. I missed it as well (last bullet point).

```
The following fields, which were previously an AccountID or AccountID*, are now a MuxedAccount or MuxedAccount* (respectively):

    PaymentOp::destination
    PathPaymentStrictReceiveOp::destination
    PathPaymentStrictSendOp::destination
    Operation::sourceAccount
    Operation::destination (for ACCOUNT_MERGE)
    Transaction::sourceAccount
    SimplePaymentResult::destination
```